### PR TITLE
Progress tab: Add translations

### PR DESCRIPTION
**Pull request by PriyangaPKini**
**Story card:** [sc-9710](https://app.shortcut.com/simpledotorg/story/9710/add-string-translations)

This batch of translations contains 
- Translations for strings in the progress tab
- Translations for strings in the notifications

- [x] Fixed the top-level keys

### CODE REVIEW

File: config/locales/api/sid_ET.yml [hunk 1]
```diff
@@ -111,11 +111,11 @@ sid-ET:
         chapter_7_html: "7: Hassiisseeme sonubba"
         chapter_8_html: "8: Duucha woyte Xa'mantino xa'mubba"
         chapter_9_html: "9: Hiittoonni kaa’lo afira dandiinayiro"
-        chapter_10_html: "10: Simple horonsirateyi hanafi"
+        chapter_10_html: "10: Shota horonsirateyi hanafi"
```
Comment (PriyangaPKini) [inline config/locales/api/sid_ET.yml:114]: Do we want to localize the word `Simple`?

Comment (prabhanshuguptagit) [reply config/locales/api/sid_ET.yml:114]: Let's ignore these changes. You can ask @harimohanraj89 to help flag this to GPI (our translations agency) so they keep this in mind in the future.

**Comment by prabhanshuguptagit** [comment]
The PR looks good :ok: Let's see if we can get these reviewed by someone, I've asked [here](https://simpledotorg.slack.com/archives/C03URFX2LUT/p1669985602822579?thread_ts=1669896949.884989&cid=C03URFX2LUT).

Can you check locally on this branch that these translations show up on progress tab?

File: config/locales/api/hi_IN.yml [hunk 2]
```diff
@@ -315,3 +315,116 @@ hi-IN:
...
+        controlled_card:
+          subtitle: "पिछले 3 माह में अपनी पिछली विज़िट पर %{controlled_threshold} वाले >3 माह में रजिस्टर किए गए %{facility_name} पर %{diagnosis} रोगी।"
+          help_tooltip:
+            numerator: "पिछले 3 माह में अपनी पिछली विज़िट पर %{controlled_threshold} वाले रोगी।"
+            denominator: "पिछले 3 माह से पहले रजिस्टर किए गए %{diagnosis} रोगी %{facility_name} को असाइन किए गए। फॉलो-अप रोगियों में मृत और लावारिस शव को बाहर रखा गया था।"
+        uncontrolled_card:
+          subtitle: "पिछले 3 माह में अपनी पिछली विज़िट पर %{uncontrolled_threshold} वाले >3 माह में रजिस्टर किए गए %{facility_name} पर %{diagnosis} रोगी।"
+          help_tooltip:
+            numerator: "पिछले 3 माह में अपनी पिछली विज़िट पर %{uncontrolled_threshold} वाले रोगी।"
+            denominator: "पिछले 3 माह से पहले रजिस्टर किए गए %{diagnosis} रोगी %{facility_name} को असाइन किए गए। फॉलो-अप रोगियों में मृत और लावारिस शव को बाहर रखा गया था।"
```
[the remark this answers is not in the document]
Comment (PriyangaPKini) [reply config/locales/api/hi_IN.yml:410]: Sent an email to GPI regarding the same

**Pull request merged**